common/src/main/java/com/microsoft/identity/common/internal/controllers/BrokerMsalController.java

```diff
@@ -236,14 +236,22 @@ public AcquireTokenResult acquireToken(final @NonNull InteractiveTokenCommandPar
         //Get the broker interactive parameters intent
         final Intent interactiveRequestIntent = getBrokerAuthorizationIntent(parameters);
 
+        // To support calling from OneAuth-MSAL, which may be initialized without an Activity
+        // add Flags to start as a NEW_TASK if we are launching from an application Context
+        final boolean startAsNewTask = null == parameters.getActivity();
```

Comment:
nit: Why create a flag here and branch on it later?  parameters.getActivity() == null is a pretty compelling reason not to call .startActivity on it.  Moving it down also brings the comment into context with the addFlags call.

Reply:
1e7863706333d0a7edc5105ef172e77a2a575539